uplc-semantics.md

```diff
@@ -3,13 +3,15 @@
 ```k
 require "uplc-builtins.md"
 require "uplc-discharge.md"
+require "uplc-genvironment-instance.md"
```

Comment:
From my understanding we're basically loading this static global environment in all cases and all uses. I understand that this solution reduces the size of the environment when proving claims about large programs but is there a way to do this specifically for proving real-world contracts? I'm not sure if it's a good idea to force regular `krun` and small proofs to adopt the global environment that you added here. Maybe you can leave the requires clauses that look in the global environment but maybe you could initialize the global environment to an empty one except when running real-world contracts.

Reply:
Maybe one thing to consider is to load the instance of the global environment in the verification module?